mol_dqn/chemgraph/dqn/deep_q_networks.py

```diff
@@ -197,7 +197,7 @@ def _build_single_q_network(self, observations, head, state_t, state_tp1,
     # the asymptotic relative efficiency of the l1 loss estimator is better
     # for heavy-tailed distributions.
     errors = tf.where(
-        tf.abs(td_error) < 1.0, tf.square(td_error),
+        tf.abs(td_error) < 1.0, tf.square(td_error) * 0.5,
```

Comment:
tf.losses.huber_loss?

Reply:
Note that your code doesn't _exactly_ match the huber loss right now; you have a strict inequality instead of <=

Reply:
Resolved; this is equivalent.